### PR TITLE
Main/feature

### PR DESCRIPTION
**Pull request by jamestw13**
Updated Main.java
Functionality and issues are listed in Main.java description

### CODE REVIEW

File: Main.java [hunk 1]
```diff
@@ -1,237 +1,448 @@
...
 		grid.setHgap(5);
 		grid.setPadding(new Insets(10,10,10,10));
+
+
+
+		mainFoodList = new FoodData();
+		mainFoodList.loadFoodItems("foodItems.txt");
+
+		tempFoodList = FXCollections.observableArrayList(mainFoodList.getAllFoodItems());
+//        tempFoodList.addListener(new ListChangeListener<FoodItem>() {
```
Comment (nisler) [inline Main.java]: What is this commented out section for? Can you get rid of it?

File: Main.java [hunk 2]
```diff
@@ -1,237 +1,448 @@
...
-		
-		TextArea mealAnalysis = new TextArea();
-		GridPane.setConstraints(mealAnalysis, 3, 4);
-		
-		
-		grid.getChildren().addAll(loadFileButton, foodItemInsert, foodListTable, 
-				menuListTable, transferButton, queries, clearQuery, analyze, mealAnalysis);
 
-		Scene mainScene = new Scene(grid);		
+		//		TextArea mealAnalysis = new TextArea();
```
Comment (nisler) [inline Main.java]: Remove comments?

File: Main.java [hunk 3]
```diff
@@ -1,237 +1,448 @@
...
+		insertName.clear();
+		insertCals.clear();
+		insertCarbs.clear();
+		insertFat.clear();
+		insertProtein.clear();
+		insertFiber.clear();
+	}
+
+	private void nameQuery() {
+		tempFoodList = FXCollections.observableArrayList(mainFoodList.filterByName(nameText.getText()));
```
Comment (nisler) [inline Main.java]: Won't you want to call the filterByName() method that exists on the FoodData object in order to get the filtered list?

File: Main.java [hunk 4]
```diff
@@ -1,237 +1,448 @@
...
+		grid.getChildren().addAll(listMenu, foodItemInsert, foodListTable,
+				menuListTable, transferButton, queries, clearQuery, analyze);
+
+		mainScene = new Scene(grid);
 		mainStage.setScene(mainScene);
 		mainStage.show();
 
 	}
 
+	private void loadFile(){
```
Comment (nisler) [inline Main.java]: Will you want to hook into the loadFoodItems() method in the FoodData object?

File: Main.java [hunk 5]
```diff
@@ -1,237 +1,448 @@
...
+		if (file != null) {
+			mainFoodList.loadFoodItems(file.toString());
+		}
+
+		else {
+			System.out.println("File not found");
+		}
+	}
+
+	private void saveFile() {
```
Comment (nisler) [inline Main.java]: Will you want to hook into the saveFoodItems() method in the FoodData object?

File: Main.java (hunk 1 above)
Comment (jamestw13) [reply Main.java]: Commented section is work-in-progress code so tables will update automatically when changes are made like querying.

File: Main.java (hunk 2 above)
Comment (jamestw13) [reply Main.java]: Removed

File: Main.java (hunk 3 above)
Comment (jamestw13) [reply Main.java]: filterByName method is used in line 419

File: Main.java (hunk 4 above)
Comment (jamestw13) [reply Main.java]: loadFoodItems() is used on line 376.

File: Main.java (hunk 5 above)
Comment (jamestw13) [reply Main.java]: saveFoodItems() is used on line 389.

**Comment by jamestw13** [comment]
I'm new to using git. I was under the impression that a pull request could be made on code with incremental improvements. I didn't know that it was reserved for only finished products.

I'll get back to you.

File: Main.java (hunk 4 above)
Comment (jamestw13) [reply Main.java]: And it's functional - as functional as loadFoodItems() allows.

File: Main.java (hunk 5 above)
Comment (jamestw13) [reply Main.java]: And it's functional

**Pull request closed without merging**